framework/src/main/java/org/checkerframework/framework/util/ContractsUtils.java

```diff
@@ -418,16 +418,37 @@ private AnnotationMirror getAnnotationMirrorOfMetaAnnotation(
         for (Pair<AnnotationMirror, AnnotationMirror> r : declAnnotations) {
             AnnotationMirror anno = r.first;
             AnnotationMirror metaAnno = r.second;
-            List<String> expressions =
-                    AnnotationUtils.getElementValueArray(anno, "value", String.class, false);
-            AnnotationMirror postcondAnno = getAnnotationMirrorOfMetaAnnotation(metaAnno, anno);
-            if (postcondAnno == null) {
-                continue;
-            }
-            for (String expr : expressions) {
-                result.add(new Postcondition(expr, postcondAnno, anno));
+            try {
+                // If multiple contracts, then this will throw an exception which is catched at Line
+                // 433
```

Comment:
Line numbers are never useful in source code, because they will become out of date as the code before them is edited.